src/components/Sponsor/Section/Sponsors.styles.ts

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
```

Comment:
Just a sugestion, what if we changed it to SponsorsContainer? I think would make clear that it's not a wrapper for the whole component, but for only for the Sponsors component

```suggestion
export const SponsorsContainer = styled.div`
```